test/e2e/testnet/testnet.go

```diff
@@ -437,20 +482,20 @@ func (t *Testnet) StartNodes(ctx context.Context) error {
 	return nil
 }
 
-func (t *Testnet) Start(ctx context.Context) error {
+func (t *Testnet) Start(ctx context.Context, indices ...int) error {
 	// start nodes and setup proxies
-	err := t.StartNodes(ctx)
+	err := t.StartNodes(ctx, indices...)
 	if err != nil {
 		return err
 	}
 	// wait for nodes to sync
 	log.Info().Msg("waiting for genesis nodes to sync")
-	err = t.WaitToSync(ctx)
+	err = t.WaitToSync(ctx, indices...)
 	if err != nil {
 		return err
 	}
 
-	return t.StartTxClients(ctx)
+	return t.StartTxClients(ctx, indices...)
 }
```

Comment:
Why was this added?

Reply:
not sure, looks like a sanity check https://github.com/celestiaorg/celestia-app/blob/6c70dac7ce849046c8e7dbbb32f68614ef523cb9/test/e2e/testnet/testnet.go#L428-L435

cc @staheri14 

went ahead and removed in 61a747baeff12a779ac54d5f8d8fb50ebc57e398